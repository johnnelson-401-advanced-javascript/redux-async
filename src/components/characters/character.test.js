import React from 'react';
import { shallow } from 'enzyme';
import Character from './character';


describe('Character Display component', () => {
  it('renders a character display card', () => {
    const wrapper = shallow(<Character name={'name'} photoUrl={'nope'} />);
    expect(wrapper).toMatchSnapshot();
    
  });
  
});
