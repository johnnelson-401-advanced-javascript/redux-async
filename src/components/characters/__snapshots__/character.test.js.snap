// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character Display component renders a character display card 1`] = `
<figure
  className="characterFigure"
>
  <img
    alt="name"
    src="nope"
  />
  <figcaption>
    name
  </figcaption>
</figure>
`;
